src/serializer/ResidualFunctions.js

```diff
@@ -186,6 +186,39 @@ export class ResidualFunctions {
       }
     };
 
+    // Emit code for ModifiedBindings for residual functions
+    for (let [funcValue, funcInfo] of this.additionalFunctionValueInfos) {
+      // Add binding updates to additional functions
+      for (let [, residualBinding] of funcInfo.modifiedBindings) {
+        let scope = residualBinding.scope;
+        // TODO #989: This should probably be an invariant once captures work properly
+        if (!residualBinding.referentialized) continue;
+
+        // Find the place to emit statements to
+        let preludeOverride = rewrittenAdditionalFunctions.get(funcValue);
+        let prelude = overriddenPreludes.get(preludeOverride);
```

Comment:
Can you comment the meaning of key, value of "overriddenPreludes" map? It seems to map from "preludeOverride" to some kind of "prelude" which makes no sense to me. There are so many words "prelude" here.